user/user.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+"fmt"
+"html/template"
+"net/http"
+)
+
+func main(){
+	http.HandleFunc("/changeuser", createUser)
+	http.HandleFunc("/", serveHTML)
+	http.ListenAndServe(":8080", nil)
+}
+
+func createUser(w http.ResponseWriter, r *http.Request){
+	r.ParseForm()
+	firstname := r.FormValue("firstname")
+	surname := r.FormValue("surname")
+	email := r.FormValue("email")
+
+	if surname == ""{
+		errorMessage := "Error: Surname cannot be empty"
+		displayErrorPage(w, errorMessage)
+		return
+	}
+	//inserted a code to create the user using the provided data
+	fmt.Printf("User created - Frist Name: %s, Surname: %s, Email: %s\n",firstname, surname, email)
```

Comment:
when user is created this should be displayed on the html page in a corresponding message block rather than just output as a plain text